### ISSUE TITLE
Changing width of column

### ISSUE THREAD

**Issue by riannazandbelt** [riannazandbelt/more-than-just-datum#2]
Hi @mackenzierose,

I've nearly finished this assignment however, I'm encountering one issue with the three smallest screen sizes. In the table with all the data, it appears that the width of the columns for my screenshots are too far left. Therefore, I'm trying to format the columns to make the widths more narrow to resolve the issue I'm having with Markbot. I tried using <colgroup width="80"> to format the columns, but Markbot wouldn't allow me to use it. Therefore, I'm wondering if you had any suggestions as to how I could fix this? I've attached a screen with the differences between mine and the reference to hopefully better explain the issue I'm encountering.

Thanks,
Rianna

<img width="319" alt="Screen Shot 2020-03-25 at 9 04 28 AM" src="https://user-images.githubusercontent.com/55094049/77539540-ec217280-6e77-11ea-811d-8162b3e8ef21.png">

**Comment by MacKenzieRose** [comment]
It looks to me like your font sizes are too large, so I would try adjusting that!

**Comment by MacKenzieRose** [comment]
Not sure where my other comment went, but the font size looks off to me, so I would adjust that.

If it's too far to the left, increase your font size, and that should shift it over.

**Comment by riannazandbelt** [comment]
@MacKenzieRose 

Initially, it was at the default body copy font size, and when I added a class to increase or decrease the font size (mega or milli) it made the screen shots way off. I was able to tweak things so now it's just my 320px and 400px that are off by 14% and 15% respectively and they need to be less than 13% different. Any other changes you recommend I make to adjust column width?

Thanks in advance,
Rianna

<img width="404" alt="Screen Shot 2020-03-25 at 9 27 50 AM" src="https://user-images.githubusercontent.com/55094049/77541341-eed19700-6e7a-11ea-8aad-17aed6f14aaa.png">
<img width="394" alt="Screen Shot 2020-03-25 at 9 28 00 AM" src="https://user-images.githubusercontent.com/55094049/77541348-f133f100-6e7a-11ea-92f7-e9d20efe61ca.png">

<img width="316" alt="Screen Shot 2020-03-25 at 9 28 26 AM" src="https://user-images.githubusercontent.com/55094049/77541405-0577ee00-6e7b-11ea-9fd2-6b307155f6d1.png">
<img width="314" alt="Screen Shot 2020-03-25 at 9 28 39 AM" src="https://user-images.githubusercontent.com/55094049/77541419-090b7500-6e7b-11ea-99b2-b0b05e6fc13d.png">
